locomotion-sdk/src/pages/Login/AuthLoadingScreen.js

```diff
@@ -35,16 +39,16 @@ const AuthLoadingScreen = ({ navigation }) => {
         }
 
         const userData = response;
-        const userProfile = {
-          firstName: userData.firstName,
-          lastName: userData.lastName,
-          avatar: userData.avatar,
-          email: userData.email,
-          pushToken: userData.pushToken,
-          pushUserId: userData.pushUserId,
-        };
+        let cards = null;
+        try {
+          cards = await usePayments.getPaymentMethods();
+        } catch (e) {
+          console.log(e);
+        }
+
+        userData.cards = cards;
 
-        await saveUser(userProfile);
```

Comment:
sure? Do we use this data else where? I'm sure we do

Reply:
still saved, but i didn't see any reason we were only saving those fields instead of the full user